dotnet/Templates/Microsoft.MacCatalyst.Templates/maccatalyst-view/.template.config/localize/templatestrings.es.json

```diff
@@ -1,6 +1,6 @@
 {
   "author": "Microsoft",
-  "name": "Vista de Mac\u00A0Catalyst",
+  "name": "Vista de Mac Catalyst",
```

Comment:
@tj-devel709 the non-breaking space is being removed again

Reply:
I'll go ahead and remove that file change from this PR

Reply:
Okay removed from this PR!